docs/ref/models/querysets.txt

```diff
@@ -2399,6 +2399,47 @@ Class method that returns an instance of :class:`~django.db.models.Manager`
 with a copy of the ``QuerySet``’s methods. See
 :ref:`create-manager-with-queryset-methods` for more details.
 
+
+``explain()``
+~~~~~~~~~~~~~
+
+.. versionadded: 2.1
+
+.. method:: explain(verbose=False, format=None)
+
+Returns the execution plan of the current queryset as a string. This details
+how the database would execute the given query, highlighting any indexes or
+joins used. For example, when using ``postgresql``::
+
+    >>> print(Blog.objects.filter(title="My Blog").explain())
+    Seq Scan on blog  (cost=0.00..35.50 rows=10 width=12)
+      Filter: (title = 'My Blog'::bpchar)
+
+Some databases (like MySQL and Postgres) accept flags that can return more
+information about the query. These flags can be passed as case-insensitive keyword
```

Comment:
"case-insensitive" is probably irrelevant as (hopefully) most people will write `.explain(verbose=True)` and not `.explain(VERBOSE=True)`

Reply:
Good point, removed